grand/geo/coordinates.py

```diff
@@ -1283,4 +1283,4 @@ def grandcs_to_ltp(self, ltp):
 
 # RK TODO: Rework on this class.
 class Rotation(_Rotation):
-    pass
+    pass
```

Comment:
I dont undertand why this is singled out by git, but i've checked and the files are identical (in dev and dev_snonis). Note however that this file has been updated in dev_sim2root to allow for +/- 0-360 input in azimuth.